### ISSUE TITLE
generateHTML option for fallback image width

### ISSUE THREAD

**Issue by szegheo** [11ty/eleventy-img#102]
Currently, the URL picked for the fallback image is always the lowest size of the desired output widths. In situations like generating small images for mobile, this makes the fallback image blurry in old browsers like IE11.

`generateHTML` already has an `options = {}` parameter, this could be used to pass a new option like `fallbackWidth` or `fallbackMinWidth` to solve this. Maybe when `null` (original width) is in the `widths` (output widths) array we could use that by default.

**Comment by zachleat** [comment]
I believe this behavior changed in a recent version, can you double check that you're on the latest?

**Comment by coderars** [comment]
> I believe this behavior changed in a recent version, can you double check that you're on the latest?

Sure, I'm on 0.9.0.

**Comment by coderars** [comment]
@zachleat looking at the source code I think the [reason for this issue is here](https://github.com/11ty/eleventy-img/blob/master/generate-html.js#L52):
````js
if(!lowsrc || !lowsrc.length) {
  throw new Error(`Could not find the lowest <img> source for responsive markup for ${originalSrc}`);
}

attributes.src = lowsrc[0].url; // <- first item is always the image with lowest width
````
To fix this, using the last item by default would be better, and if `fallbackWidth` option (my suggestion) is defined the item with equal/closest width could be used (to not force hdpi/retina images to IE11).

**Comment by zachleat** [comment]
Just so I’m clear, this is a feature specific to support browsers that don’t support `<picture>` right?

https://caniuse.com/picture

Dropping this into the enhancement queue

> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy-img/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by zachleat** [comment]
Marginally related to #129

**Comment by zachleat** [comment]
Either shipped with #63, a duplicate of #129, or no longer relevant as `<picture>` is pretty ubiquitous now? Going to remove from the queue unless anyone objects!
